sorting/shellSort.php

```diff
@@ -0,0 +1,27 @@
+<?php
+function shell_Sort($my_array)
+{
+	$x = round(count($my_array)/2);
+	while($x > 0)
+	{
+		for($i = $x; $i < count($my_array);$i++){
+			$temp = $my_array[$i];
+			$j = $i;
+			while($j >= $x && $my_array[$j-$x] > $temp)
+			{
+				$my_array[$j] = $my_array[$j - $x];
+				$j -= $x;
+			}
+			$my_array[$j] = $temp;
+		}
+		$x = round($x/2.2);
+	}
+	return $my_array;
+}
+ 
+$test_array = array(3, 0, 2, 5, -1, 4, 1);
+echo "Original Array :\n";
+echo implode(', ',$test_array );
+echo "\nSorted Array\n:";
+echo implode(', ',shell_Sort($test_array)). PHP_EOL;
+?>
```

Comment:
Per the [PSR-12 php style guide](https://www.php-fig.org/psr/psr-12/), the closing ?> tag must be omitted from files containing only PHP

```suggestion

```